RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceBootstrappers.cpp

```diff
@@ -67,7 +67,8 @@ bool URH_GameInstanceServerBootstrapper::GetCommandlineBootstrapModeOverride(ERH
 {
 	// note - mirror changes to this function below in Initialize, which provides more error handling
 	FString BootstrapCommandlineModeString;
-	if (FParse::Value(FCommandLine::Get(), TEXT("rhbootstrapmode="), BootstrapCommandlineModeString))
+	if (FParse::Value(FCommandLine::Get(), TEXT("rhbootstrapmode="), BootstrapCommandlineModeString)
```

Comment:
I'm definitely cool with deprecating these command line options that are not prefixed by `rh.`.  Can we add a log line whenever these options are used to warn about the deprecation and to swap to their `rh.` counterparts?